### ISSUE TITLE
Update to 1.8.9

### ISSUE THREAD

**Issue by MoonlightSystem** [DarkholmeTenk/TardisModIssues#458]
it would be great if this mod updated to 1.8.9 seeing that the RF API has been ported officially to it, as well as Thaumcraft  and many other mods

**Comment by DarkholmeTenk** [comment]
1.9 is out, though, 1.8.9 is already officially outdated.

**Comment by MoonlightSystem** [comment]
1.9 forge isnt even out yet.

**Comment by DarkholmeTenk** [comment]
Nope, but the amount of work (combined with my slowness) it would take to port to 1.8.9 means forge would almost certainly be out by the time it's done

**Comment by ghost** [comment]
Do not worry we only have 557 more errors and rejects to go before forge is released to 1.9.
https://twitter.com/lexmanos

**Comment by DarkholmeTenk** [comment]
Yeah, but keep in mind they've fixed like 500 in a day or two :P

**Comment by 1skandar** [comment]
1.9 probably won't be stable enough to update to until at least 2017. Remember, there was 450 days between 1.8, which was crap, and 1.8.9, which mods are now actually updating to. So even though 1.9 is officially here, there is no reason to call 1.8.9 dead or outdated just yet.

**Comment by FoxPotato** [comment]
Wasn't 1.4.7 like the main modding version till end 1.5 too?

**Comment by 1skandar** [comment]
Pretty much. And 1.7.10 didn't start to get really popular until about the time 1.8 was released. That is generally how it works.

**Comment by FoxPotato** [comment]
I mean, this is up to dark. But if it's easy to update from 1.8 to 1.9, why not. Just an updated version, and then new updates will be 1.9 or something.

**Comment by 1skandar** [comment]
Not only that, but there is an increasing, shift to 1.8.9 and away from 1.7.10.

**Comment by Webber88** [comment]
sadly this is true

**Comment by DarkholmeTenk** [comment]
It's a lot of work and I'm exceptionally lazy, but if FP wants to update darkcore and then update TM, he can go ahead :P

**Comment by FoxPotato** [comment]
You know what, I could always try.
nothing ventured, nothing gained, amirite

**Comment by DarkholmeTenk** [comment]
https://www.reddit.com/r/feedthebeast/comments/4ah7sv/so_heres_the_deal_with_cofh/
This is a very good summation of the issue, updating is essentially going through all the code you've wrote and rewriting huge chunks of it. It's very disheartening to have to fix huge amounts of code. The only reason this is for 1.7.10 and not 1.6.4 is because I did that update a month or two into development, not 16 months in.
